website/templates/emails/transactional.html.mako

```diff
@@ -1,431 +1,418 @@
-														</td>
-                                                    </tr>
-                                                    <tr>
-                                                    	<td align="center" valign="top" style="padding-top:0;">
-                                                        	<table border="0" cellpadding="15" cellspacing="0" class="templateButton">
-                                                            	<tr>
-                                                                	<td valign="middle" class="templateButtonContent">
-                                                                    	<div>
-                                                                        	<a href="http://www.mandrill.com/" target="_blank">View on the OSF</a>
-                                                                        </div>
-                                                                    </td>
-                                                                </tr>
-                                                            </table>
-                                                        </td>
-                                                    </tr>
-                                                </table>
-                                                <!-- // End Module: Standard Content \\ -->
-
-                                            </td>
-                                        </tr>
-                                    </table>
-                                    <!-- // End Template Body \\ -->
+                        </table>
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                       <table id="content" width="90%" border="0" cellpadding="0" cellspacing="0" align="center">
+                           <tr> 
+                            <td> 
+                               <p class="lead text-center"> Hello ${name}, there has been recent activity on your OSF projects!</p>
+                                <div class="comment-block div-center">
+                                    ${message}                                
+                                </div>  
+                               </td>
+                           </tr> 
+                        </table>
+                   </td>
+                </tr>
+                <tr>
+                    <td>
+                        <table width="80%" border="0" cellpadding="10" cellspacing="0" align="center">
+                            <tr>
+                                <td>
+                                    <div class="line"></div>
+                                    <p class="small text-center">Copyright &copy; 2015 Center For Open Science, All rights reserved. </p>
+                                    <p class="small text-center"><a href="#">View this email in a browser </a> |  <a href="#">update subscription preferences</a></p> 
```

Comment:
@caneruguz, could you add in the links? The `url` context variable should be used for "update subscription preferences"